rpm-armhf/Dockerfile

```diff
@@ -39,6 +39,11 @@ RUN echo "armhfp" > /etc/yum/vars/basearch && \
 RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 && \
     rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-AltArch-Arm32
 
+# Centos 7 EOLed on July 1st, 2024. We need to switch to vault.centos.org to get the packages
+RUN sed -i s/mirror.centos.org/vault.centos.org/g /etc/yum.repos.d/*.repo && \
```

Comment:
I think the used mirror is `mirrorlist.centos.org`

Reply:
Indeed, but this URL is only used in the `mirrorlist` section that gets disabled

The other required change (AFAIK) is to replace `baseurl=mirror.centos.org` by `baseurl=vault.centos.org`

Reply:
Ah yes you're right!